lib/features/parkings/parkings_view/widgets/parking_favourite.dart

```diff
@@ -11,11 +11,11 @@ class FavouriteParkingWidget extends ConsumerWidget {
   final Parking parking;
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final favController = ref.watch(localFavParkingsRepositoryProvider(parking.id).notifier);
     final isFavorite = ref.watch(localFavParkingsRepositoryProvider(parking.id));
+
     return IconButton(
       visualDensity: VisualDensity.compact,
-      onPressed: favController.toggle,
+      onPressed: ref.read(localFavParkingsRepositoryProvider(parking.id).notifier).toggle,
```

Comment:
```suggestion
      onPressed: (){
        ref.read(localFavParkingsRepositoryProvider(parking.id).notifier).toggle
     },
```

it doesn't work the same Tomuś :)